build.gradle.kts

```diff
@@ -32,6 +32,10 @@ dependencies {
   testImplementation(libs.kotest)
 }
 
+kotlin {
```

Comment:
i think we should set it in `buildSrc/src/main/kotlin/intellijbsp.kotlin-conventions.gradle.kts`

Reply:
Make sense, updated.